File: app/src/main/java/ng/com/techdepo/esidemtest/models/Options.java
package ng.com.techdepo.esidemtest.models;

import com.google.gson.annotations.SerializedName;

import java.io.Serializable;

public class Options implements Serializable {

    @SerializedName("a")
    String optionA;
    @SerializedName("b")
    String optionB;
    @SerializedName("c")
    String optionC;
    @SerializedName("d")
    String optionD;

    public String getOptionA() {
        return optionA;
    }

    public void setOptionA(String optionA) {
        this.optionA = optionA;
    }

    public String getOptionB() {
        return optionB;
    }

    public void setOptionB(String optionB) {
        this.optionB = optionB;
    }

    public String getOptionC() {
        return optionC;
    }

    public void setOptionC(String optionC) {
        this.optionC = optionC;
    }

    public String getOptionD() {
        return optionD;
    }

    public void setOptionD(String optionD) {
        this.optionD = optionD;
    }
}

File: app/src/main/java/ng/com/techdepo/esidemtest/models/Options.kt
package ng.com.techdepo.esidemtest.models

import com.google.gson.annotations.SerializedName

import java.io.Serializable

class Options : Serializable {

    @SerializedName("a")
    var optionA: String?=null
    @SerializedName("b")
    var optionB: String?=null
    @SerializedName("c")
    var optionC: String?=null
    @SerializedName("d")
    var optionD: String?=null

    constructor()

    constructor(optionA: String, optionB: String, optionC: String, optionD: String) {
        this.optionA = optionA
        this.optionB = optionB
        this.optionC = optionC
        this.optionD = optionD
    }
}
